components/member-profile/index.js

```diff
@@ -8,10 +8,157 @@ const Profile = (props) => {
   } = props;
   const memberName = `${first_name || ''} ${last_name || ''}`;
   return (
-    <>
-      <img src={imageLink} className={classNames.profilePic} alt="profile" />
-      <div className={classNames.memberName}>{memberName.trim() || '--'}</div>
-    </>
+    <div className={classNames.container}>
+      {/* <img src={imageLink} className={classNames.profilePic} alt="profile" />
+      <div className={classNames.memberName}>{memberName.trim() || '--'}</div> */}
+
+      <div className={(classNames.sidebar, classNames.column)}>
+        <div className={classNames.memberDetails}>
+          <img src={imageLink} className={classNames.profilePic} alt="ProfilePicture" />
+          <div className={classNames.personalInfo}>
+            <h1 className={classNames.profileName}>{memberName.trim() || '--'}</h1>
+            <p className={classNames.userName}>@userName</p>
+            <p className={classNames.workDetails}>
+              FrontEnd Developer
+              <br />
+              <span className={classNames.userName}>WayCool.in</span>
+            </p>
+          </div>
+          <div className={classNames.socialIcons}>
+            <img
+              src="https://staging-members-rds.herokuapp.com/icons/instagram.png"
+              alt="Instagram"
+            />
+            <img
+              src="https://staging-members-rds.herokuapp.com/icons/linkedin.png"
+              alt="LinkedIn"
+            />
+            <img src="https://staging-members-rds.herokuapp.com/icons/github.png" alt="GitHub" />
```

Comment:
NIT : the src links , we can store this in a const variables at the top and then use it here
example : `<img src={instagramImg} alt="Instagram">`
The same goes with LinkedIn, Github , Twitter, etc

Reply:
Sure, Will make the changes.

Reply:
@sumitd94 Changes are done, can you please approve?